server/routes/index.js

```diff
@@ -23,9 +29,11 @@ router.post('/login', login);
 router.post('/brand', isAuth, isAdmin, postBrand);
 router.post('/type', isAuth, isAdmin, postType);
 
+router.put('/update/car/:carId', isAuth, isAdmin, updateCarControllers);
+
 router.get('/brands', getBrands);
 router.get('/logout', logout);
-router.delete('/admin/deleteCar/carId', isAuth, isAdmin, deleteCar);
+router.delete('/admin/deleteCar/:carId', isAuth, isAdmin, deleteCar);
```

Comment:
also its better like that
```suggestion
router.delete('/admin/cars/:carId', isAuth, isAdmin, deleteCar);
```